config/initializers/theme_loader.rb

```diff
@@ -18,13 +18,13 @@ def require_theme(theme_name)
 
   require theme_main_include
 
-  Rails.configuration.paths.add(
-    'config/refusal_advice',
-     with: root.join('config/refusal_advice'),
-     glob: '*.yml'
+  Rails.configuration.paths['config/refusal_advice'].push(
+    root.join('config/refusal_advice')
```

Comment:
I'm assuming the glob added on L26 also applies to the theme path because they're the same "key" (`'config/refusal_advice'`)?

Reply:
Yeah as I understand it, line 26 is setting up a configuration path which can exist in core but doesn't need to - and we don't have this path. This path can also have alternative paths which is what we add when loading the theme or in specs.

Rails doesn't have a way of doing `Rails.configuration.paths.remove` but it can add/remove these alternative paths which is what is done in the spec after callback hook using `unshift`.

Reply:
Cool makes sense 👍